### PR TITLE
Add Blog Announcing Conference

### PR DESCRIPTION
**Pull request by jorydotcom**
Closes #<issue number>

### CODE REVIEW

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "🎉 Announcing GraphQLConf 2024"
+tags: ["blog"]
+layout: blog
+date: 2024-03-28
+byline: GraphQL Conference & Event Committee
+permalink: /blog/2024-03-28-announcing-graphqlconf-2024
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
```
no longer used I'll clean up soon all blogpost

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "🎉 Announcing GraphQLConf 2024"
+tags: ["blog"]
+layout: blog
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
```
too

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+### 📅 Mark Your Calendars
+
+* **CFP Deadline:** May 22, 2024, at 11:59 pm PDT
+* **Early Bird Registration Ends:** May 22, 2024
+* **Schedule Announcement:** June 26, 2024
+* **Event Dates:** September 10-12, 2024
+
+### 🤝 Get Ready to Connect and Collaborate
+
+[GraphQLConf 2024](https://graphql.org/conf/2024/) is not just another tech conference; it's a gathering of diverse and vibrant community members who are passionate about pushing the boundaries of GraphQL and building the future of APIs together. Whether you're a seasoned GraphQL user, or just looking to understand how GraphQL can help your business, GraphQLConf is a can’t-miss opportunity to be part of this ecosystem.
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
[GraphQLConf 2024](/conf/2024) is not just another tech conference; it's a gathering of diverse and vibrant community members who are passionate about pushing the boundaries of GraphQL and building the future of APIs together. Whether you're a seasoned GraphQL user, or just looking to understand how GraphQL can help your business, GraphQLConf is a can’t-miss opportunity to be part of this ecosystem.
```
to benefit client side routing

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+### 🔭 Navigating API Innovation
+
+For technology leaders navigating the complexities of modern API architectures, GraphQLConf 2024 is more than just a conference. It's a strategic platform to foresee industry trends, meet the pioneers, and evaluate how GraphQL can fit into and enhance your technology strategy. Whether you're assessing GraphQL for your next project, looking to optimize existing infrastructures, or keen to foster innovation within your team, GraphQLConf promises a wealth of knowledge and networking opportunities to help steer your decisions in the right direction.
+
+### 📢 Call for Proposals (CFP) is OPEN!
+
+Do you have a big idea, a transformative project, or an insightful case study? We want you on our stage! [The CFP is officially open](https://graphql.org/conf/2024/), and we’re on the lookout for speakers who can bring fresh perspectives and valuable knowledge to our community. Don't miss this chance to showcase your expertise and contribute to the collective learning of the GraphQL ecosystem. Submit your proposal now—CFP closes on May 22, 2024, at 11:59 pm PDT.
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
Do you have a big idea, a transformative project, or an insightful case study? We want you on our stage! [The CFP is officially open](/conf/2024), and we’re on the lookout for speakers who can bring fresh perspectives and valuable knowledge to our community. Don't miss this chance to showcase your expertise and contribute to the collective learning of the GraphQL ecosystem. Submit your proposal now—CFP closes on May 22, 2024, at 11:59 pm PDT.
```

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+title: "🎉 Announcing GraphQLConf 2024"
+tags: ["blog"]
+layout: blog
+date: 2024-03-28
+byline: GraphQL Conference & Event Committee
+permalink: /blog/2024-03-28-announcing-graphqlconf-2024
+---
+
+Inviting innovators, decision-makers, and GraphQL aficionados alike! GraphQL Foundation are thrilled to announce the return of [GraphQLConf](https://graphql.org/conf/2024/), the much-anticipated official conference for anyone working with GraphQL. After a very successful inaugural conference last year, we are setting the stage for an even bigger event, returning once again to San Francisco — but this time to the heart of this vibrant city! Mark your calendars for September 10-12, 2024, as we invite you to join us at the stunning JW Marriott in San Francisco Union Square for a three-day celebration of learning, innovation, networking, and community.
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
Inviting innovators, decision-makers, and GraphQL aficionados alike! GraphQL Foundation are thrilled to announce the return of [GraphQLConf](/conf/2024), the much-anticipated official conference for anyone working with GraphQL. After a very successful inaugural conference last year, we are setting the stage for an even bigger event, returning once again to San Francisco — but this time to the heart of this vibrant city! Mark your calendars for September 10-12, 2024, as we invite you to join us at the stunning JW Marriott in San Francisco Union Square for a three-day celebration of learning, innovation, networking, and community.
```

File: website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+* **CFP Deadline:** May 22, 2024, at 11:59 pm PDT
+* **Early Bird Registration Ends:** May 22, 2024
+* **Schedule Announcement:** June 26, 2024
+* **Event Dates:** September 10-12, 2024
+
+### 🤝 Get Ready to Connect and Collaborate
+
+[GraphQLConf 2024](https://graphql.org/conf/2024/) is not just another tech conference; it's a gathering of diverse and vibrant community members who are passionate about pushing the boundaries of GraphQL and building the future of APIs together. Whether you're a seasoned GraphQL user, or just looking to understand how GraphQL can help your business, GraphQLConf is a can’t-miss opportunity to be part of this ecosystem.
+
+We can't wait to see you in San Francisco! Stay tuned for more updates, and don't forget to check out our conference page [here](https://graphql.org/conf/2024/) for the latest news and announcements.
```
Comment (dimaMachina) [inline website/src/pages/blog/2024-03-28-announcing-graphqlconf-2024.mdx]:
```suggestion
We can't wait to see you in San Francisco! Stay tuned for more updates, and don't forget to check out our conference page [here](/conf/2024) for the latest news and announcements.
```

**Review by benjie — APPROVED** [review]

**Comment by dimaMachina** [comment]
deployment fails since I changed build settings, can you merge with `source` branch?
image

**Comment by jorydotcom** [comment]
HECKYAH

**Pull request merged**